src/commands/play.rs

```diff
@@ -116,6 +119,12 @@ pub async fn _play(
 
 async fn calculate_time_until_play(queue: &[TrackHandle], flag: &PlayFlag) -> Option<Duration> {
     if !queue.is_empty() {
+        let non_livestreams = queue.iter().filter_map(|track| track.metadata().duration);
+
+        if non_livestreams.count() < queue.len() {
+            return Some(Duration::MAX);
+        }
```

Comment:
This is not true for `/playtop` right?

Reply:
Yeah, you're right. Oversight by my part, it'll show "Estimated time until play: ∞" which is incorrect.